playbooks/roles/edxapp/defaults/main.yml

```diff
@@ -702,6 +702,17 @@ EDXAPP_SESSION_SAVE_EVERY_REQUEST: false
 
 EDXAPP_SESSION_COOKIE_SECURE: false
 
+EDXAPP_INSTRUCTOR_QUERY_PROBLEM_TYPES:
+  - 'problem',
+  - 'openassessment',
+  - 'submit-and-compare',
+  - 'freetextresponse',
+  - 'xblockmufi',
+  - 'qualtricssurvey',
```

Comment:
@stvstnfrd touched base with monica, and we switched qualtrics survey with the survey xblock which is very similar to the poll xblock